tools/rum/rum-slicer.css

```diff
@@ -930,8 +957,8 @@ facet-sidebar[aria-disabled="true"] div.quick-filter {
   main .key-metrics ul {
     grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
   }
-}
 
-footer.footer-wrapper.appear {
-  display: none;
+  footer.footer-wrapper.appear {
+    display: none;
+  }
```

Comment:
The footer should be hidden always. That is the reason for the issue.

Reply:
Hi @vdua , seems it was part of lint issues fixes that I pushed.
This will be fixed in my upcoming commits. thanks.